<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>980</width>
    <height>588</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QRadioButton" name="csv_radio_button">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">display: flex;
align-items: center;
ustify-content: center;
cursor: bold;
padding: 0px 15px;
line-height: 34px;
border: 1px solid #999;
border-radius: 6px;
user-select: none;
background: #945bf0;
</string>
    </property>
    <property name="text">
     <string>CSV файл</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="excel_radio_button">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>20</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">display: flex;
align-items: center;
ustify-content: center;
cursor: bold;
padding: 0px 15px;
line-height: 34px;
border: 1px solid #999;
border-radius: 6px;
user-select: none;
background: #945bf0;
</string>
    </property>
    <property name="text">
     <string>xls файл</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="sql_radio_button">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>20</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">display: flex;
align-items: center;
ustify-content: center;
cursor: bold;
padding: 0px 15px;
line-height: 34px;
border: 1px solid #999;
border-radius: 6px;
user-select: none;
background: #945bf0;
</string>
    </property>
    <property name="text">
     <string>SQL</string>
    </property>
   </widget>
   <widget class="QPushButton" name="open_file_button">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>481</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">display: inline-block;
cursor: pointer;
padding: 0px 15px;
line-height: 34px;
border: 1px solid #999;
border-radius: 6px;
user-select: none;
background: #9f99a8</string>
    </property>
    <property name="text">
     <string>Выбрать файл</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>80</y>
      <width>441</width>
      <height>341</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="column_comboBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>190</y>
      <width>481</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">display: inline-block;
cursor: pointer;
padding: 0px 15px;
line-height: 34px;
border: 1px solid #999;
border-radius: 6px;
user-select: none;
background: #9f99a8</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>160</y>
      <width>261</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Выберете столбец для работы с ним</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>270</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Мода</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="modeline">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>300</y>
      <width>481</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">display: inline-block;
cursor: pointer;
padding: 0px 15px;
line-height: 34px;
border: 1px solid #999;
border-radius: 6px;
user-select: none;
background: #fffff</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>340</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Медиана</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="medianline">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>370</y>
      <width>481</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">display: inline-block;
cursor: pointer;
padding: 0px 15px;
line-height: 34px;
border: 1px solid #999;
border-radius: 6px;
user-select: none;
background: #fffff</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>410</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Среднее</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="avgline">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>440</y>
      <width>481</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">display: inline-block;
cursor: pointer;
padding: 0px 15px;
line-height: 34px;
border: 1px solid #999;
border-radius: 6px;
user-select: none;
background: #fffff</string>
    </property>
   </widget>
   <widget class="QComboBox" name="pass_comboBox">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>500</y>
      <width>301</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">display: inline-block;
cursor: pointer;
padding: 0px 15px;
line-height: 34px;
border: 1px solid #999;
border-radius: 6px;
user-select: none;
background: #fffff</string>
    </property>
   </widget>
   <widget class="QPushButton" name="passButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>500</y>
      <width>151</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">display: inline-block;
cursor: pointer;
padding: 0px 15px;
line-height: 34px;
border: 1px solid #999;
border-radius: 6px;
user-select: none;
background: #fffff</string>
    </property>
    <property name="text">
     <string>Заполнить пропуски</string>
    </property>
   </widget>
   <widget class="QPushButton" name="InButton">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>10</y>
      <width>221</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">display: inline-block;
cursor: pointer;
padding: 0px 15px;
line-height: 34px;
border: 1px solid #999;
border-radius: 6px;
user-select: none;
background: #fffff</string>
    </property>
    <property name="text">
     <string>Войти или зарегестрироваться</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>80</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">display: inline-block;
cursor: pointer;
padding: 0px 15px;
line-height: 34px;
border: 1px solid #999;
border-radius: 6px;
user-select: none;
background: #fffff</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>231</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Разделитель(только для csv файлов)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>240</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">display: inline-block;
cursor: pointer;
padding: 0px 15px;
line-height: 34px;
border: 1px solid #999;
border-radius: 6px;
user-select: none;
background: #9f99a8</string>
    </property>
    <property name="text">
     <string>Вычислить</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelError">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>510</y>
      <width>431</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>980</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
